src/csharp/Grpc.Core/Internal/NativeLogRedirector.cs

```diff
@@ -51,6 +51,7 @@ public static void Redirect(NativeMethods native)
             }
         }
 
+        [MonoPInvokeCallback(typeof(GprLogDelegate))]
```

Comment:
If this is not added, then this runtime error occurs:

> Attempting to JIT compile method '(wrapper native-to-managed) Grpc.Core.Internal.NativeLogRedirector:HandleWrite (intptr,int,ulong,intptr,intptr)' while running in aot-only mode. See https://developer.xamarin.com/guides/ios/advanced_topics/limitations/ for more information.

Reply:
Actually, scratch that, it looks like the MonoPInvokeCallbackAttribute is defined lower in the file and the AOT goes just by the name of that attribute.